compiler/mio-tflite260/include/mio_tflite260/Util.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __MIO_TFLITE260_UTIL_H__
+#define __MIO_TFLITE260_UTIL_H__
+
+#include <mio/tflite/schema_generated.h>
+
+namespace mio
+{
+namespace tfl
```

Comment:
other libraries used `tflite` not `tfl`

Reply:
I tried to modify this as `tflite` but because `schema_generated.h` is already using `namespace tflite`, I failed :(
So I will remove `namespace tfl` and just only leave it as `mio` :)

Reply:
> is already using namespace tflite, I failed :(

For me, just `mio` is also ok.
I', not sure what may be the problem, but just to check the failure, can you try with `::mio::tflite` from the caller?

Reply:
Did you mean following trial?

```cpp
namespace mio
{

tflite::BuiltinOperator builtin_code_neutral(const tflite::OperatorCode *opcode);
// ...

} // namespace mio

mio::tflite::builtin_code_neutral(opcode); // COMPILE FAILED
mio::builtin_code_neutral(opcode); // COMPILE & RUNNING SUCCESS
```

---

FYI, original trial was following :(
```cpp
namespace mio
{
namespace tflite
{

tflite::BuiltinOperator builtin_code_neutral(const tflite::OperatorCode *opcode);
// ...

} // namespace tflite
} // namespace mio
```

```sh
Helper.h:27:9: error: ‘BuiltinOperator’ in namespace ‘mio::tflite’ does not name a type
 tflite::BuiltinOperator builtin_code_neutral(const tflite::OperatorCode *opcode);
         ^~~~~~~~~~~~~~~
...
```
`tflite::BuiltinOperator` is interpreted as `mio::tflite::BuiltinOperator`

Reply:
> It failed because tflite::BuiltinOperator is interpreted as mio::tflite::BuiltinOperator

```cxx
namespace mio
{
namespace tflite
{

::tflite::BuiltinOperator builtin_code_neutral(const tflite::OperatorCode *opcode);
// ...

} // namespace tflite
} 
```
maybe try with `::tflite::BuiltinOperator` instead of `tflite::BuiltinOperator`

anyway, using `mio` instrad of `tflite` looks OK.

one thing, if you are to revise `mio-circle` too, then `mio::tfltie` and `mio::circle` would look better.

Reply:
> maybe try with ::tflite::BuiltinOperator instead of tflite::BuiltinOperator

I think it would work but not sure how much should I change those things :)
So as you advised, when I try to revise `mio-circle` later, I will think together this!